dojo/templates/dojo/simple_search.html

```diff
@@ -83,9 +83,17 @@ <h2> {{ name }}</h2>
                         </thead>
                         <tbody>
                         {% for product in products %}
+                            {% comment %} sometimes the index contains references to findings that no longer exist. even after rebuilding the index. {% endcomment %}
+                            {% if product.object.id %}                        
                             <tr>
-                                <td><a class="search-finding"
-                                       href="{% url 'view_product' product.object.id %}">{{ product.object.name }}</a>
+                                <td>
+                                    {% comment %} it appears sometimes the finding id is missing from the search index, even after rebuilding the watson index {% endcomment %}
+                                    {% if product.object.id %}
+                                        <a class="search-finding"
+                                            href="{% url 'view_product' product.object.id %}">{{ product.object.name }}</a>
+                                    {% else %}
+                                        VALENTIJN{{ product.object.name }}
```

Comment:
Is this an easter egg? :)

Reply:
haha nice find. looks like my cherry-picking and squashing skills are still on holiday